persistence/module.go

```diff
@@ -183,7 +183,7 @@ func (m *persistenceModule) NewRWContext(height int64) (modules.PersistenceRWCon
 }
 
 func (m *persistenceModule) NewReadContext(height int64) (modules.PersistenceReadContext, error) {
-	conn, err := connectToDatabase(m.config.GetPostgresUrl(), m.config.GetNodeSchema())
+	conn, err := connectToDatabase(m.config, m.config.GetNodeSchema())
```

Comment:
If we're going to be passing in the whole config, then there's no need for a second parameter.